### PR TITLE
Discord badge in Readme

### PR DESCRIPTION
**Pull request by agriffard**
Should display the Discord badge in Readme with number of users online.

### CODE REVIEW

**Comment by agriffard** [comment]
https://shields.io/badges/discord

File: README.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ Orchard Core consists of two distinct projects:
 [![BSD-3-Clause License](https://img.shields.io/badge/license-BSD--3--Clause-blue.svg)](LICENSE)
 [![Documentation](https://readthedocs.org/projects/orchardcore/badge/)](https://docs.orchardcore.net/)
 [![Crowdin](https://badges.crowdin.net/orchard-core/localized.svg)](https://crowdin.com/project/orchard-core)
+[![Discord](https://img.shields.io/discord/551136772243980291?color=%237289DA&label=OrchardCore&logo=discord&logoColor=white&style=flat-square)](https://orchardcore.net/discord)
```
Comment (Piedone) [inline README.md:13]: Shouldn't this already work? 
{4BCD193A-D2C3-4EE0-95B0-B47F8C013B4C}

**Review by Piedone — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (agriffard) [reply README.md:13]: It seems an admin has to enable a setting: 
https://github.com/badges/shields/issues/1659#issuecomment-384814358

Comment (Piedone) [reply README.md:13]: Got it, enbled, now it works.

**Pull request merged**